index.html

```diff
@@ -291,15 +291,17 @@ <h3>
         </p>
       </section>
     </section>
-    <section>
+    <section data-cite="MEDIAQUERIES-5 manifest-incubations">
       <h2>
-        Addition of `window-controls-overlay` to the manifest
+        Addition of <code><dfn data-export="">window-controls-overlay</dfn>
+        </code> to the manifest
       </h2>
       <p>
         To enable the WCO feature, a new value will be added to the
```

Comment:
In the first paragraph, define the display mode here, similar to what you see in here:
https://www.w3.org/TR/mediaqueries-5/#display-mode

Then in the second paragraph, mention that developers can specify this display mode in the [=display_override=] field in the manifest.

I think that should cover us?

Reply:
Done